src/app/page.tsx

```diff
@@ -1,5 +1,5 @@
 export const metadata: Metadata = {
-  title: "jmm.limwa.pt",
```

Comment:
I'm we could use an environment variable here to override this and then fallback to "jmm compiler". Something like APP_NAME should be good!

Reply:
I thought of that as well, just decided not to do it because I don't know how environment variables would interact with the next.js build process

Reply:
This is the relevant documentation for page metadata https://nextjs.org/docs/app/api-reference/functions/generate-metadata.

I think just using `process.env.APP_NAME ?? "jmm compiler"` in the title will just work, as long as the env variable is available during next build!

Reply:
Done!